xlators/protocol/client/src/client-handshake.c

```diff
@@ -1370,7 +1370,7 @@ client_handshake(xlator_t *this, struct rpc_clnt *rpc)
     if (!frame)
         goto out;
 
-    req.gfs_id = 0xbabe;
+    req.gfs_id = 0xcafe;
```

Comment:
It's unclear to me where this value is ever used.

Reply:
Yeah, I didn't find any use of it but went up with the second option in the issue `to at least set to appropriate value`. I feel we can remove the `gfs_id` from all the places:

rpc/rpc-lib/src/rpcsvc.h
xlators/mgmt/glusterd/src/glusterd-handshake.c
xlators/protocol/client/src/client-handshake.c

Reply:
It can't be deleted without breaking protocol. So don't remove it.

Reply:
Oh ok. Thanks @pranithk :)

Reply:
This value is used in the communication protocol, so it's not so easy to remove it (it could cause backward incompatibilities). However it's true that it doesn't seem to be used anywhere. I haven't found any reason for it.

I would keep it, but I would use another value here. `0xcafe` is already used in the glusterd handshake (xlators/mgmt/glusterd/src/glusterd-handshake.c). To be able to distinguish them in case it's needed sometime, I would use another value.

Reply:
@xhernandez Ack.